.github/workflows/ci.yml

```diff
@@ -51,6 +51,8 @@ jobs:
           - test-modify-latest
           - test-out-of-order
           - test-rename
+          - test-conflicting-duplicate
+          - test-conflicting-out-of-order
```

Comment:
There were two kinds of conflicts not being accounted for:
1. when a merge leads to migrations with duplicated versions
2. when the PR branch is behind main, and the merge would introduce out-of-order migrations